utils/ccloud_library.sh

```diff
@@ -836,3 +836,42 @@ function ccloud::destroy_ccloud_stack() {
 
   return 0
 }
+
+##############################################
+# These are some duplicate functions from 
+#  helper.sh to decouple the script files.  In 
+#  the future we can work to remove this 
+#  duplication if necessary
+##############################################
+function ccloud::retry() {
+    local -r -i max_wait="$1"; shift
+    local -r cmd="$@"
+
+    local -i sleep_interval=5
+    local -i curr_wait=0
+
+    until $cmd
+    do
+        if (( curr_wait >= max_wait ))
+        then
+            echo "ERROR: Failed after $curr_wait seconds. Please troubleshoot and run again."
+            return 1
+        else
+            printf "."
+            curr_wait=$((curr_wait+sleep_interval))
+            sleep $sleep_interval
+        fi
+    done
+    printf "\n"
+}
+function ccloud::version_gt() { 
+  test "$(printf '%s\n' "$@" | sort -V | head -n 1)" != "$1";
+}
+function ccloud::check_expect() {
```

Comment:
In this library, all functions that started with `check` were intentionally converted to `validate`.  If this function is being brought into this library, it also has to change to follow the other functions that validate a bin is installed.  So:

`check_expect` --> `validate_expect_installed`